resources/views/components/chirp/chirp-comment.blade.php

```diff
@@ -1,10 +1,13 @@
-<div>
-    <p class="leading-5 text-sm text-gray-600">{{ $comment->user->name }} chirps: {{ $comment->content }}</p>
+<div id="comment-{{$comment->id}}">
+    <div>
+        <p class="leading-5 text-sm text-gray-600">{{ $comment->user->name }} chirps: {{ $comment->content }}</p>
+    </div>
+    @if ($comment->user->is(auth()->user()))
+        <form class="comment-delete-form">
```

Comment:
add the action attribute back, and then instead of declaring the url in the javascript, get the url via this.action or form.action